### DISCUSSION TITLE
html

### DISCUSSION THREAD

**Discussion by Hassib1234** [CSproMates/to-do-app#47] [General]
i not know  the html

**Comment by notghettolenny** [comment]
check out this resource
https://www.codecademy.com/learn/learn-html

**Comment by Hassib1234** [comment]
مننه

**Comment by ghost** [comment]
Or you can use this website https://www.freecodecamp.org/learn/responsive-web-design/

**Comment by IgorSwatowski** [comment]
Check it out first. 

https://www.w3schools.com/html/

**Comment by JoaoLCom** [comment]
this video can help you 
https://www.youtube.com/watch?v=qz0aGYrrlhU

**Comment by RosaliaLey** [comment]
Hi, I recommend taking the free course on Frontend Masters. https://frontendmasters.com/courses/web-development-v2/
